packages/sdks/snippets/react/src/components/CustomColumns.tsx

```diff
@@ -0,0 +1,39 @@
+import { Blocks, BuilderBlock } from '@builder.io/sdk-react';
+
+interface CustomColumnsProps {
+  columns: { blocks: BuilderBlock[] | undefined }[];
+  builderBlock: { id: string | undefined };
```

Comment:
you can use `BuilderBlock` here:

```suggestion
  builderBlock: BuilderBlock;
```